composer.json

```diff
@@ -8,26 +8,26 @@
     "require": {
         "drupal/core": "~8.6.0@alpha",
```

Comment:
This version restriction seems very limiting since it will not allow to go to 8.7.0. Since we are only days away from the final release of Drupal 8.6, can we maybe change this to `~8.6`? This should normally work since our minimum stability is set to `dev`.